Package.swift

```diff
@@ -31,6 +31,7 @@ let package = Package(
         // Dependencies declare other packages that this package depends on.
         .package(url: "https://github.com/IBM-Swift/Kitura-NIO.git", from: "2.0.0"),
         .package(url: "https://github.com/IBM-Swift/BlueCryptor.git", from: "1.0.0"),
+        .package(url: "https://github.com/IBM-Swift/BlueSSLService.git", from: "1.0.0"),
```

Comment:
Is this just for testing? It's kind of annoying... Any way we can avoid it?

Reply:
Yes, for testing only. The `HTTPServer.sslConfig` is of type `SSLService.Configuration`

Reply:
😢